rust/src/measurements/gumbel_max/make_report_noisy_max_gumbel.tex

```diff
@@ -1,253 +1,248 @@
+ 
+\begin{proof} 
+\begin{align*} 
+    \ln\left(\frac{\sum_{i} \exp(\frac{\epsilon x'_i}{\Delta})}{\sum_{i} \exp(\frac{\epsilon x_i}{\Delta})}\right) 
+    &= \ln\left(\frac{\sum_{i} \exp(\frac{\epsilon (x'_i - x_i + x_i)}{\Delta})}{\sum_{i} \exp(\frac{\epsilon x_i}{\Delta})}\right) \\ 
+    &= \ln\left(\frac{\sum_{i} \exp(\frac{\epsilon (x'_i - x_i)}{\Delta})\exp(\frac{\epsilon (x_i)}{\Delta})}{\sum_{i} \exp(\frac{\epsilon x_i}{\Delta})}\right) \\ 
+    &\le \ln\left(\frac{\exp(\frac{\epsilon \max_j(x'_j - x_j)}{\Delta}) \sum_{i} \exp(\frac{\epsilon x_i}{\Delta})}{\sum_{i} \exp(\frac{\epsilon x_i}{\Delta})}\right) \\ 
+    &= \frac{\epsilon \max_j(x'_j - x_j)}{\Delta} 
+\end{align*} 
+\end{proof} 
+ 
+\label{privacy-guarantee} 
+Assume $x$, $x'$ in \texttt{input\_domain} are \din-close under \rustdoc{metrics/struct}{LInfDistance} and $\texttt{privacy\_map}(\din) \le \dout$. 
+Let the output random variables be denoted $Y \sim \function(x)$ and $Y' \sim \function(x')$. 
+ 
+\begin{align*} 
+    &\max_{x \sim x'} D_\infty(Y || Y') \\ 
+    \leq& \max_{x \sim x'} \max_i \ln\left(\frac{\Pr[\function(x) = i]}{\Pr[\function(x') = i]}\right)  
+    &&\text{by } \rustdoc{measures/struct}{MaxDivergence}\\ 
+    =& \max_{x \sim x'} \max_i \ln\left(\frac 
+            {\Pr[\mathrm{argmax}_k (x_k / \tau - \ln(-\ln(U_k))) = i]} 
+            {\Pr[\mathrm{argmax}_k (x'_k / \tau - \ln(-\ln(U_k))) = i]} 
+        \right) &&\text{by \ref{function-prob}, substitute \function}\\ 
+    =& \max_{x \sim x'} \max_i \ln \left( 
+        \frac 
+            {\exp \frac{x_i}{\tau}} 
+            {\sum_{k} \exp\frac{x_k}{\tau} 
+        } 
+        \bigg/ \frac 
+            {\exp\frac{x'_i}{\tau}} 
+            {\sum_{k} \exp\frac{x'_k}{\tau}} \right) 
+        &&\text{by \ref{gumbel-reparam}} \\ 
+    =& \max_{x \sim x'} \max_i \ln \left(\frac 
+        {\exp\frac{x_i}{\tau}}{\exp\frac{x'_i}{\tau}} 
+        \frac{\sum_{k} \exp\frac{x'_k}{\tau}}{\sum_{k} \exp\frac{x_k}{\tau}}\right) \\ 
+    =& \max_{x \sim x'} \max_i \ln \left(\frac 
+        {\exp\frac{x_i}{\tau}}{\exp\frac{x'_i}{\tau}}\right) + \ln \left( 
+        \frac{\sum_{k} \exp\frac{x'_k}{\tau}}{\sum_{k} \exp\frac{x_k}{\tau}}\right) \\ 
+    =& \max_{x \sim x'} \frac{\max_i (x_i - x'_i)}{\tau}  
+        + \ln\left(\frac{\sum_{k} \exp\frac{x'_k}{\tau}}{\sum_{k} \exp\frac{x_k}{\tau}}\right) \\ 
+    \leq& \max_{x \sim x'} \frac{\max_i (x_i - x'_i)}{\tau} + \frac{\max_j (x'_j - x_j)}{\tau} &&\text{by \ref{priv-inequality}} \\ 
+    \leq& \frac{\max_{x \sim x'} \max_{ij} \abs{(x_i - x'_i) - (x_j - x'_j)}}{\tau} \\ 
+    =& d_{in} / \tau &&\text{by }\texttt{RangeDistance}\\ 
+\end{align*} 
+% If monotonic, $\Delta = \din$, else $\Delta = 2 \cdot \din$.  
+% Now substitute the inequality that $ \tau \geq \Delta / \epsilon$. 
+ 
+Since this expression aligns with the pseudocode given for the privacy map, 
+it has been shown that $\function(x)$ and $\function(x')$ are \dout-close under \texttt{output\_measure}  
```

Comment:
It has been shown where? Cite the theorem and the paper.

Reply:
Edited to reference the align above.